cmd/root.go

```diff
@@ -46,6 +49,18 @@ func NewRootCmd() *cobra.Command {
 		Short: "proxy-init adds a Kubernetes pod to the Linkerd service mesh",
 		Long:  "proxy-init adds a Kubernetes pod to the Linkerd service mesh.",
 		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if options.TimeoutCloseWaitSecs != 0 {
+				sysctl := exec.Command("sysctl", "-w",
+					fmt.Sprintf("net.netfilter.nf_conntrack_tcp_timeout_close_wait=%d", options.TimeoutCloseWaitSecs),
+				)
+				out, err := sysctl.CombinedOutput()
+				fmt.Println(string(out))
```

Comment:
Can you use `log.Println` like in `iptables.go`? So that in both places we output to stderr.